Makefile.minimal

```diff
@@ -1,7 +1,8 @@
 # Makefile to build the SDL library
 
 INCLUDE = -I./include
-CFLAGS  = -g -O2 $(INCLUDE)
+CFLAGS ?= -g -O2
+CFLAGS += $(INCLUDE)
```

Comment:
Perhaps, we can move the `-I` to `CPPFLAGS`? The `-I` is meant for the preprocessor after all.
```suggestion
CPPFLAGS = -I./include
CFLAGS = -g -O2
```

Reply:
Indeed; I tend to avoid suggesting `CPPFLAGS` because I’ve had rather violent reactions to that in the past, but SDL already uses `CPPFLAGS` extensively so it should be fine here.